docker-cmd-windows.sh

```diff
@@ -0,0 +1,179 @@
+#!/bin/bash
+
+# Example of docker usage for RLLIB + SUMO Utlis
+#
+# Author: Lara CODECA
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# See https://github.com/lcodeca/rllibsumodocker/blob/master/docker-cmd.sh
+# for the original.
+
+set -e
+
+IMAGE_NAME="tf-gpu-sumo-$(date +%Y-%m-%d)"
+IMAGE_FOLDER="docker-image"
+GPU=true
+GPU_OPT="--gpus all"
+OPTIRUN=false
```

Comment:
optirun does not exist for Mac OSX

* https://wiki.archlinux.org/index.php/Bumblebee